### DISCUSSION TITLE
Adding more layers in instrument editor?

### DISCUSSION THREAD

**Discussion by richiholland** [hydrogen-music/hydrogen#1538] [General]
Is there a way to add more layers to the instrument editor? Currently there's 16. Would like to add more for round-robins and such. 

If not, is this a feature that could be added in future release? Or, is there a way I could modify the code to enable more layers?

**Comment by elpescado** [comment]
Hi, maximum number of layers is already configurable in preferences:

<img width="775" alt="hydrogen-layers" src="https://user-images.githubusercontent.com/1619113/159166668-72a23ae7-ac6b-4638-a412-700e64eb0555.png">

**Reply by richiholland** [reply]
OMG, I missed that. Thank you!
